fleet-tenancy/README.md

```diff
@@ -0,0 +1,167 @@
+   configure resources.
+2. The source of truth is created. If not, use the public [example](config)
+   as a quickstart.
+3. A fleet host project already exists.
+4. Use ACM version 1.17.0 or later.
+
+## 1. [Set up your fleet]
+
+This step includes the following operations:
+- Create a Fleet in a project
+- Enable API services
+- Create a service account and grant the service account a list of IAM roles.
+
+Terraform configs: [link](terraform/1-fleet).
+
+Apply the resources via Terraform: [README.md](terraform/1-fleet/README.md).
+
+Notes:
+- You may need to run this command multiple times to get all resources created
+  successfully due to internal dependencies.
+- After the step, you need to create and download the service account’s private
+  key JSON file.
+
+## 2. Configure fleet-level defaults for Config Sync
+
+This step configures fleet-level defaults to install Config Sync on entire
+fleet, which syncs [fleet tenancy configs](config) to all clusters in the fleet.
+
+Fleet tenancy configs include the following:
+- Two NamespaceSelector objects with the [dynamic mode], one for the `frontend`
+  team scope, and the other one for the `backend` team scope. In the dynamic
+  mode, NamespaceSelector selects both statically-declared Namespaces and those
+  dynamically present on the clusters with matching labels. For more details,
+  see [Limit which namespaces a config affects].
+- One `ResourceQuota` object that is synced to all [fleet namespaces] of the
+  team scope `frontend`.
+- One `NetworkPolicy` object that is synced to all fleet namespaces of the team
+  scope `backend`.
+
+Terraform configs: [link](terraform/2-fleet-default-config).
+
+Apply the resources via Terraform: [README.md](terraform/2-fleet-default-config/README.md).
+
+Notes:
+- The Fleet Default Member Config must be created before clusters are registered 
+  to the fleet. The default config won’t apply to previously registered clusters.
+
+## 3. Create and register clusters
+
+This step creates and registers clusters. For simplicity, we use GKE clusters
+and register them for the same project as the fleet.
+
+Terraform configs: [link](terraform/3-clusters).
+
+Apply the resources via Terraform: [README.md](terraform/3-clusters/README.md).
+
+## 4. Configure Fleet Scopes and Namespaces
+
+This step includes the following operations:
+- Create team scopes: `backend` and `frontend`
+- Add clusters to scopes
+- Create fleet Namespaces
+
+Terraform configs: [link](terraform/4-scopes-and-namespaces).
+
+Apply the resources via Terraform: [README.md](terraform/4-scopes-and-namespaces/README.md).
+
+## 5. Sync more resources to the scope
+
+All scopes share the same source of truth. If you want to sync more sources, you
+can add the configurations to the source repository with the correct
+NamespaceSelector annotation. Config Sync will automatically reconcile those
+resources.
+
+## 6. Remove a cluster from a scope
```

Comment:
I'd suggest removing step 6 and 7 to keep it concise. Instead, have a paragraph that says the resources will be synced dynamically to fleet namespaces in team scopes, when new clusters is added to or removed from a scope, or when a fleet namespace is added or removed.

Reply:
done